fdmi/fol_fdmi_src.c

```diff
@@ -776,8 +776,8 @@ m0_fol_fdmi_filter_kv_substring(struct m0_conf_fdmi_filter   *filter,
 		if (fol_frag->rp_ops->rpo_type != &m0_fop_fol_frag_type)
 			continue;
 		fop_fol_frag = fol_frag->rp_data;
-		if (fop_fol_frag->ffrp_fop_code != M0_CAS_PUT_FOP_OPCODE)
-			continue;
+		//if (fop_fol_frag->ffrp_fop_code != M0_CAS_PUT_FOP_OPCODE)
+		//	continue;
```

Comment:
Why is this needed?
Do you mean, we want to have all events? (just for demo)

Reply:
yes, why not? And there are not "ALL", just "all", because there are only put and del.

Reply:
This is a demo app.
Why do we need to change this to catch all (put and del) operations?
If we need to catch put and del operations, this checking can be removed.

Here. just commenting out the code seems to be a hacking. It just creates some confusion.

Also, no C++ comments.

Reply:
ok, disabled it for now. Can be fixed as part of another change-set.

Reply:
OK